peopleops/compensation.md

```diff
@@ -0,0 +1,32 @@
+# Compensation
+
+Total compensation consists of 3 elements, salary, equity, and benefits.
+FlowForge aims to provide a total compensation package on par with similar
+companies in terms of: stage of the company, market, role, and level of the role.
+
+## Salary
+
+Salary for each role is determined initially during the hiring process based
+on compensation expectation of the candidate as well as the market rates. Salary
+is based on your location you live in and the company context. FlowForge pays in
+your local currency unless explicitly discussed otherwise.
+
+### Adjustments
+
+Salary can be increased twice a year. In September, a performance adjustment is
+done. Performance review is based on values, and meeting role expectations.
+
+Each December compensation is adjusted for each employee with the company for
+over 6 months. Inflation is set to the year average for the country the employee
```

Comment:
Yearly average seems to me like the wrong number to use. It should be something like 
`Inflation is set to the value for the preceding 12 months as measured by a recognised national statistics authority in the employees country` 
You can't use an average of monthly values as inflation is measured between 2 dates. Given that the salary is adjusted annually we should use the period since the last adjustment.

Reply:
In that case we should move this item to January and pick the annual number as reported by each government. Hopefully there's some aggregate source to mitigate checking for each national authority, like the IMF or so.

I'll try to investigate how to properly write this down, and push a commit tomorrow. Thanks for the feedback @sammachin